PROJECT

```diff
@@ -1,16 +1,44 @@
+domain: metal3.io
+layout:
+- go.kubebuilder.io/v3
 multigroup: true
```

Comment:
Its there check the [migration guide](https://book.kubebuilder.io/migration/multi-group.html)

Reply:
if you are talking about the below part, that's for v2
```
version: "2"
domain: tutorial.kubebuilder.io
repo: tutorial.kubebuilder.io/project
multigroup: true
```

Reply:
all the available fields are mentioned in https://book.kubebuilder.io/reference/project-config.html

Reply:
I followed https://book.kubebuilder.io/migration/migration_guide_v2tov3.html#migrate-the-apis , and that suggests the same as it was for v2. The fields was added by kubebuilder v3. I am reluctant to remove it specially since it is added by kubebuilder v3 itself. 

`Please run kubebuilder edit --multigroup=true to enable multi-group support before migrating the APIs and controllers. Please see [this](https://book.kubebuilder.io/migration/multi-group.html) for more details.`

Reply:
Ok, I got confused then. Because in the Kubebuilder book, they have not mentioned multigroup field, while it seems it is possible to add that field.